### PR TITLE
add check imagestreams as composite action

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

**Comment by zmiklank** [comment]
Depends on: https://github.com/sclorg/ci-scripts/pull/121

**Review by phracek — CHANGES_REQUESTED** [review]
See my comment.

File: .github/workflows/check-imagestreams.yml [hunk 1]
```diff
@@ -7,48 +7,17 @@ on:
...
-          sha=$(git rev-parse HEAD)
-          result="success"
-          python ci-scripts/ocp-stream-generator/ocp-stream-generator/stream_generator.py
-          git diff --exit-code -- imagestreams || result="failure"
-          echo "result=$result" >> "$GITHUB_OUTPUT"
-          echo "sha=$sha" >> "$GITHUB_OUTPUT"
-
-      - name: Set final commit status
-        uses: myrotvorets/set-commit-status-action@v2.0.0
+      - uses: sclorg/ci-scripts/ocp-stream-generator@master
```
[the remark this answers is not in the document]
Comment (zmiklank) [reply .github/workflows/check-imagestreams.yml:21]: in that case we can add this action as part of openshift tests.

[the remark this answers is not in the document]
Comment (zmiklank) [reply .github/workflows/check-imagestreams.yml]: fixed by: https://github.com/sclorg/nginx-container/pull/291/commits/e4a77e602aaedde0dacb0c1e7a7899afa6ee972e.

**Review by phracek — APPROVED** [review]

**Pull request merged**